docs/spec/historical.rst

```diff
@@ -188,25 +188,30 @@ When checking Python 2.7 code, type checkers should treat the ``int`` and
 type ``str`` as well as ``unicode`` should be acceptable.
 
 
-Positional-only arguments
--------------------------
+Positional-only parameters
+--------------------------
 
 Some functions are designed to take their arguments only positionally,
 and expect their callers never to use the argument's name to provide
 that argument by keyword. Before Python 3.8 (:pep:`570`), Python did
-not provide a way to declare positional-only arguments.
+not provide a way to declare positional-only parameters.
+
+To support positional-only parameters on older Python versions, type
```

Comment:
This reasoning arguably doesn't apply any more, because 3.7 is no longer supported. Still, the old syntax should remain supported for compatibility with old code. Perhaps we could add a statement like "While Python 3.7 and lower are no longer supported, type checkers should continue to support this syntax to support code written for earlier versions of Python."